tests/upgrade/run.sh

```diff
@@ -235,7 +235,7 @@ test_upgrader() {
 install_metrics_server_and_deactivate() {
     info "Install the metrics server and deactivate it to reproduce ROX-4429"
 
-    kubectl apply -f https://github.com/kubernetes-sigs/metrics-server/releases/download/v0.3.6/components.yaml
+    kubectl apply -f https://github.com/kubernetes-sigs/metrics-server/releases/download/v0.8.2/components.yaml
```

Comment:
Did you mean `v3.8.2`? This URL is returning a 404

Reply:
Or 0.6.0? I don't see a 0.8.2 release: https://github.com/kubernetes-sigs/metrics-server/releases/